tempesta_fw/sock.c

```diff
@@ -482,15 +485,39 @@ ss_tcp_process_data(struct sock *sk)
 		skb_dst_drop(skb);
```

Comment:
Maybe so, I don't mind to move it to ```ss_do_send()```. The intention to add the call here was to "forget" routing information for ingress skb and process it further the same way as locally generated skbs.